### DISCUSSION TITLE
[회고] 예진의 업무 기록:: TDD/Unit Test를 통해 함수를 쪼개는 기준 세우기 초안

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#154] [4. 기타 글 모음]
[블로그 링크](https://donkeysdevelpment.tistory.com/133)

TDD/Unit Test를 섞어서 적으면 안 되지만.. 제목은 바뀔 수도 있습니다. 그리고 TDD를 자세하게 다룰지 말지도 고민이라서 카테고리 정하는 것도 고민이 있었네요.

다음주에 참고자료 정독하면서 본문 내용 마무리해보도록 하겠습니다.
---
것보다 요즘 퇴근 후랑 주말을 너무 날리고 있어서 큰일입니다.. 생활패턴이 망가지고 졸업과제 자꾸 회피해서 큰일났어요 (비상!!!!) 네.. 졸과 조금이라도 하고 자보겠습니다. 

즐겁게 주말 마무리하세요~~

**Comment by 02ggang9** [comment]
딱 어제 TDD에 관한 내용으로 정말 고민했는데 기대가 됩니다..!
